beluga/include/beluga/sensor/likelihood_field_model.hpp

```diff
@@ -153,17 +152,30 @@ class LikelihoodFieldModel : public Mixin {
 
   /// \copydoc LaserSensorModelInterface2d::update_sensor(measurement_type&& points)
   void update_sensor(measurement_type&& points) final {
-    const auto lock = std::lock_guard<std::shared_mutex>{points_mutex_};
+    const auto lock = std::lock_guard<std::shared_mutex>{mutex_};
     points_ = std::move(points);
   }
 
+  /// \copydoc LaserSensorModelInterface2d::update_map(Map&& map)
+  void update_map(OccupancyGrid&& map) final {
+    const auto lock = std::lock_guard<std::shared_mutex>{mutex_};
+    grid_ = std::move(map);
+    free_states_ = compute_free_states(grid_);
+    likelihood_field_ = make_likelihood_field(params_, grid_);
+  }
```

Comment:
It seems that adding a test for this method should make the codecov status check green.

https://app.codecov.io/gh/Ekumen-OS/beluga/pull/189/blob/beluga/include/beluga/sensor/likelihood_field_model.hpp#L160

Reply:
Added and rebased to solve conflicts. Is there no way to run coverage checks locally?

Reply:
There isn't. I plan to add it soon. Feel free to create a ticket if you think it needs to be tracked.